src/Posts.jsx

```diff
@@ -45,68 +45,50 @@ const selectTab = (selectedTab) => {
   loadMorePosts();
 };
 
-let gatewayModeratedUsersRaw = Social.get(
-  `${moderatorAccount}/moderate/users`,
-  "optimistic",
-  {
-    subscribe: true,
-  }
-);
-
+// get the full list of posts that the current user has flagged so
+// they can be hidden
 const selfFlaggedPosts = context.accountId
   ? Social.index("flag", "main", {
       accountId: context.accountId,
-    })
-  : [];
-
-const selfModeration = context.accountId
-  ? Social.index("moderate", "main", {
-      accountId: context.accountId,
-    })
-  : [];
-
-if (gatewayModeratedUsersRaw === null) {
-  // haven't loaded filter list yet, return early
-  return "";
-}
-
-const gatewayModeratedUsers = gatewayModeratedUsersRaw
-  ? JSON.parse(gatewayModeratedUsersRaw)
+    }) ?? []
   : [];
 
-// get the full list of posts that the current user has flagged so
-// they can be hidden
-
-// expecting moderation structure for accounts and posts like
+// V2 self moderation data, structure is like:
 // { moderate: {
-//     "account1.near": "block",
+//     "account1.near": "report",
 //     "account2.near": {
-//         "100000123": "spam",
+//         ".post.main": { // slashes are not allowed in keys
+//           "100000123": "spam", // post ids are account/blockHeight
+//         }
 //     },
 //   }
 // }
-function matchesModeration(moderated, item) {
-  let accountFound = moderated[accountId];
+const selfModeration = context.accountId
+  ? Social.getr(`${context.accountId}/moderate`, "optimistic") ?? []
```

Comment:
The `?? []` is the change that fixes the feed not populating. Applied the same to flaggedPosts just to be sure.